src/main/java/quickcache/model/flashcard/Difficulty.java

```diff
@@ -0,0 +1,73 @@
+package quickcache.model.flashcard;
+
+import static java.util.Objects.requireNonNull;
+
+import quickcache.commons.util.AppUtil;
+
+public class Difficulty {
+
+    public static final String MESSAGE_CONSTRAINTS = "Difficulty names should only be "
+            + "low, medium, or high. "
+            + "No spaces allowed. Cannot be empty.";
+    public static final String VALIDATION_REGEX = "\\p{Alnum}+";
+
+    public final String difficulty;
```

Comment:
Can we name this differently to avoid misrepresentation and confusion? Think we used value for `Question` and `Answer`

Reply:
okay I will change it! thanks